### PR TITLE
Feature/getting started imx93

### PR DESCRIPTION
**Pull request by balix-microej**

### CODE REVIEW

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+
+* An Internet connection to access Github repositories & :ref:`Module Repositories <module_repositories>`.
+* MICROEJ SDK 6 (installed during :ref:`Environment Setup <sdk_6_getting_started_imx93_environment_setup>`).
+
+The second part consists of running the same demo application on your device. For that, you will need:
+
+* An i.MX93 Evaluation Kit, available `here <https://www.nxp.com/design/design-center/development-boards/i-mx-evaluation-and-development-boards/i-mx-93-evaluation-kit:i.MX93EVK>`__.
+* (Optional) An HDMI display.
+* Linux image (TBD)
+* GCC Toolchain (TBD)
```
Comment (nhouze-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:33]:
```suggestion
* Yocto Linux image (TBD)
* Yocto SDK (TBD)
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+Prerequisites
+-------------
+
+.. note::
+  
+   This Getting Started has been tested on Windows 10 & 11 with a WSL distribution Ubuntu 22.04.
+
+This Getting Started is separated into two main parts.
+
+The first part consists of running a demo application on the Virtual Device. All you need is:
```
Comment (nhouze-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:22]:
```suggestion
The first part consists in running a demo application on the Virtual Device. All you need is:
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+
+
+* An Internet connection to access Github repositories & :ref:`Module Repositories <module_repositories>`.
+* MICROEJ SDK 6 (installed during :ref:`Environment Setup <sdk_6_getting_started_imx93_environment_setup>`).
+
+The second part consists of running the same demo application on your device. For that, you will need:
```
Comment (nhouze-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:28]:
```suggestion
The second part consists in running the same demo application on your device. For that, you will need:
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+"""""""""""""""""""""""""""""
+
+The Linux image is available here: (TBD)
+
+For this getting started we use a Linux image flashed on an SD card.
+
+Linux
++++++
+
+To flash the image on Linux, use the following command:
```
Comment (nhouze-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:207]:
```suggestion
To flash the image on Linux, use the following command (assuming the SDCard device is /dev/mmcblk0):
```

File: SDK6UserGuide/gettingStarted.rst [hunk 5]
```diff
@@ -43,6 +43,9 @@ These VEE ports are ideal to evaluate MicroEJ's technology. They include the lat
    :widths: 50 50
    :header-rows: 0
 
+   * - |logo_nxp|
```
Comment (nhouze-microej) [inline SDK6UserGuide/gettingStarted.rst:46]: shouldn't we put it after rt1170 ?

Comment (balix-microej) [reply SDK6UserGuide/gettingStarted.rst:46]: As discussed: keep current order.

**Comment by cblanchard-microej** [comment]
For the Environment Setup and more specifically the Hardware Setup Part, I would have made something like that: 

- Flash the image on an SD card
   - Linux
   - Windows
- Install the Yocto SDK
- Install required packages
- Hardware Setup
First the picture of the board with the red circle (Add BMODE DIP switch)
   - Configure boot
   - LVDS display support
   - Connectivity (Take a look at what is done in RT1170 Getting Started, especially for the logs)

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+
+* Before using this task, you will have to configure the device IP Address in the environment variables.
+
+  * To do so, right-click on ``runOnDevice`` then ``Modify Run Configuration...``.
+  * Fill in your variables in ``Environment variables``, each one separated by a semicolon:
+
+    * ``SSH_USER=root``
+    * ``SSH_HOSTNAME``: IP address of the i.MX93 Evaluation Kit.
+    * ``APP_SDK_INSTALL``: Path to the Yocto SDK (by default ``/usr/local/oecore-x86_64/``).
+    * ``WSL_DISTRIBUTION_NAME``: Name of the WSL distribution if using Windows.
```
Comment (cblanchard-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:388]:
```suggestion
    * ``WSL_DISTRIBUTION_NAME``: Name of the WSL distribution if using Windows (by default ``Ubuntu``).
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 7]
```diff
@@ -0,0 +1,434 @@
...
+   Come back to this page if you need to activate an Evaluation License.
+
+* Before using this task, you will have to configure some environment variables that depend on the OS you are using.
+
+  * To do so, right-click on ``buildExecutable`` then ``Modify Run Configuration...``.
+  * Fill in your variables in ``Environment variables``, each one separated by a semicolon (see screenshot): 
+
+    * If you changed the default Yocto SDK installation path, set the following environment variable
+
+      * ``APP_SDK_INSTALL=/usr/local/oecore-x86_64`` (replace it with your own path)
```
Comment (cblanchard-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:337]:
```suggestion
      * ``APP_SDK_INSTALL``: Path to the Yocto SDK (by default ``/usr/local/oecore-x86_64/``)
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+
+    * When using Windows with WSL, set the WSL distribution name, which you can get with the command :guilabel:` wsl --list` in Windows terminal
```
Comment (cblanchard-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:339]:
```suggestion
    * When using Windows with WSL, set the WSL distribution name, which you can get with the command :guilabel:`wsl --list` in Windows terminal
```

File: SDK6UserGuide/gettingStartedIMX93.rst [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+* Before using this task, you will have to configure the device IP Address in the environment variables.
+
+  * To do so, right-click on ``runOnDevice`` then ``Modify Run Configuration...``.
+  * Fill in your variables in ``Environment variables``, each one separated by a semicolon:
+
+    * ``SSH_USER=root``
+    * ``SSH_HOSTNAME``: IP address of the i.MX93 Evaluation Kit.
+    * ``APP_SDK_INSTALL``: Path to the Yocto SDK (by default ``/usr/local/oecore-x86_64/``).
+    * ``WSL_DISTRIBUTION_NAME``: Name of the WSL distribution if using Windows.
+
```
Comment (cblanchard-microej) [inline SDK6UserGuide/gettingStartedIMX93.rst:389]: Be consistent with `buildExecutable` task and add a picture of the variables in the IntelliJ IDE

Comment (balix-microej) [reply SDK6UserGuide/gettingStartedIMX93.rst:408]: Added a screenshot

**Review by nhouze-microej — APPROVED** [review]
tested OK with latest binaries

**Pull request merged**